src/vcf-pdf-viewer.js

```diff
@@ -628,11 +628,10 @@ class PdfViewerElement extends
     }
 
     setCurrentPage(value) {
-        if (!this.__viewer || this.__loading) {
+        if (value != undefined) {
             this.$.currentPage.value = value;
-        } else {
-            this.__pageChange();
         }
+        this.__pageChange();
```

Comment:
Just a question: should `__pageChange()` be called given that `this.$.currentPage.value` might have no changes?

Reply:
The value is set in the previous call or by entering in it in the  page numbers textfield, or by using the arrows or by selecting a thumbnail. By default is the first page.

Reply:
👍